web/modules/custom/joinup_migrate/joinup_migrate.post_update.php

```diff
@@ -159,3 +161,63 @@ function joinup_migrate_post_update_more_redirects() {
     }
   }
 }
+
+/**
+ * Switches the ownership of the 'Sharing and reuse of IT solutions' collection.
+ */
+function joinup_migrate_post_update_srits_change_owner() {
+  $collection_id = 'http://data.europa.eu/w21/e5f0febe-f29c-428a-a1a2-6bb0ccd37949';
+  // The 'Joinup moderator'.
+  $current_owner_id = '700003';
+  // The 'Joinup editor'.
+  $new_owner_id = '6363';
+  /** @var \Drupal\og\MembershipManagerInterface $membership_manager */
+  $membership_manager = \Drupal::service('og.membership_manager');
+  /** @var \Drupal\Core\Entity\EntityTypeManager $entity_storage */
+  $entity_type_manager = \Drupal::service('entity_type.manager');
+
+  $collection = $entity_type_manager->getStorage('rdf_entity')->load($collection_id);
+  if (empty($collection)) {
+    throw new Exception(t('Collection with id :id was not found.', [
```

Comment:
As general rule: Do not use `t()` in exception messages. The reason is that, when we throw an exception we want to avoid side effects. Calling a function might obscure the original error cause. During exception throw we want to exit as quick as we can with a clear message. Also translation makes no sense because such messages are for developers. Using a double quote string is just fine to embed a variable in such cases: `"Collection with id $collection_id was not found."`. More, root level namespaced classes should always be called with the qualified namespace: `\Exception` (like `\Drupal`).